applications/NXxas_new.nxdl.xml

```diff
@@ -0,0 +1,230 @@
+            <doc>TODO</doc>
+            <dimensions rank="1">
+              <dim index="1" value="nEnergy" />
+            </dimensions>
+        </field>
+        <field name="intensity" type="NX_FLOAT" units="NX_ANY">
+            <doc>TODO</doc>
+            <dimensions rank="1">
+              <dim index="1" value="nEnergy" />
+            </dimensions>
+        </field>
+        <field name="intensity_errors" type="NX_FLOAT" units="NX_ANY">
+            <doc>TODO</doc>
+            <dimensions rank="1">
+              <dim index="1" value="nEnergy" />
+            </dimensions>
+        </field>
+        <group type="NXsample">
+            <field name="name">
+                <doc>Descriptive name of the sample</doc>
+            </field>
+        </group>
+        <group type="NXprocess" optional="true">
+            <doc>
+                Description on how :ref:`energy &lt;/NXxas_new/ENTRY/energy-field&gt;`
+                and :ref:`intensity &lt;/NXxas_new/ENTRY/intensity-field&gt;` were obtained
+                from the raw data.
+            </doc>
+        </group>
+        <group type="NXinstrument" optional="true">
+            <group type="NXsource">
+                <field name="type"/>
+                <field name="name"/>
+                <field name="probe">
+                    <enumeration>
+                        <item value="x-ray"/>
+                    </enumeration>
+                </field>
+            </group>
+            <group type="NXmonochromator" name="monochromator" optional="true">
+                <field name="energy" type="NX_FLOAT" units="NX_ENERGY" optional="true">
+                    <dimensions rank="1">
+                      <dim index="1" value="nEnergy" />
+                    </dimensions>
+                </field>
+                <group type="NXcrystal" name ="crystal" optional="true">
+                    <field name="d_spacing" type="NX_FLOAT" units="NX_LENGTH">
+                        <doc>spacing between crystal planes of the reflection</doc>
+                    </field>
+                    <field name="type">
+                        <doc>Type or material of monochromating substance (Si, Ge, Multilayer).</doc>
+                    </field>
+                    <field name="reflection" type="NX_INT" units="NX_UNITLESS">
+                        <doc>Miller indices (hkl) values of nominal reflection</doc>
+                        <dimensions><dim index="1" value="3"/></dimensions>
+                    </field>
+                </group>
+            </group>
+            <group type="NXdetector" minOccurs="0" maxOccurs="unbounded">
+                <field name="data" type="NX_NUMBER">
+                    <dimensions rank="1">
+                        <dim index="1" value="nEnergy" />
+                      </dimensions>
+                </field>
+            </group>
+            <group type="NXdetector" name="i0" optional="true">
```

Comment:
The i0 should have a corresponding NXmonitor group. There are also many ways of measuring an i0 that is useful to document in the file.